diff --git a/packer/build.sh b/packer/build.sh
index eb8c1d7..854c7c8 100755
--- a/packer/build.sh
+++ b/packer/build.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 region=${1-us-east-2};
-source_ami=${2-ami-8c122be9};
+source_ami=${2-ami-0cf31d971a3ca20d6};
 
 echo "AWS Region: $region"
 echo "AWS Source AMI: $source_ami"
diff --git a/packer/cassandra.json b/packer/cassandra.json
index 63cc485..0e99031 100644
--- a/packer/cassandra.json
+++ b/packer/cassandra.json
@@ -1,7 +1,7 @@
 {
   "variables": {
     "region": "us-east-2",
-    "source_ami": "ami-8c122be9",
+    "source_ami": "ami-0cf31d971a3ca20d6",
     "app_name": "Apache Cassandra 3.11.x",
     "environment": "Drift"
   },
diff --git a/packer/kafka.json b/packer/kafka.json
index cab4363..77d7f68 100644
--- a/packer/kafka.json
+++ b/packer/kafka.json
@@ -1,7 +1,7 @@
 {
   "variables": {
     "region": "us-east-2",
-    "source_ami": "ami-8c122be9",
+    "source_ami": "ami-0cf31d971a3ca20d6",
     "app_name": "Apache Kafka 2.x",
     "environment": "Drift"
   },
diff --git a/packer/opennms.json b/packer/opennms.json
index 6d2252b..02436c4 100644
--- a/packer/opennms.json
+++ b/packer/opennms.json
@@ -1,7 +1,7 @@
 {
   "variables": {
     "region": "us-east-2",
-    "source_ami": "ami-8c122be9",
+    "source_ami": "ami-0cf31d971a3ca20d6",
     "app_name": "OpenNMS Horizon 23",
     "environment": "Drift"
   },
diff --git a/packer/postgresql.json b/packer/postgresql.json
index 08988b1..ab0a203 100644
--- a/packer/postgresql.json
+++ b/packer/postgresql.json
@@ -1,7 +1,7 @@
 {
   "variables": {
     "region": "us-east-2",
-    "source_ami": "ami-8c122be9",
+    "source_ami": "ami-0cf31d971a3ca20d6",
     "app_name": "PostgreSQL 10",
     "environment": "Drift"
   },
diff --git a/packer/scripts/common.sh b/packer/scripts/common.sh
index f7e036b..567cc91 100644
--- a/packer/scripts/common.sh
+++ b/packer/scripts/common.sh
@@ -19,7 +19,12 @@ echo "### Installing common packages..."
 
 sudo yum -y -q update
 sudo yum -y -q install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-sudo yum -y -q install jq unzip net-snmp net-snmp-utils git pytz dstat htop sysstat nmap-ncat tree
+sudo yum -y -q install jq unzip git pytz tree
+
+echo "### Installing local monitoring tools..."
+
+sudo yum -y -q install net-snmp net-snmp-utils dstat htop sysstat nmap-ncat
+curl -L https://bit.ly/glances | sudo /bin/bash
 
 echo "### Configuring Net-SNMP..."
 
diff --git a/packer/scripts/kafka.sh b/packer/scripts/kafka.sh
index 3e5ed9d..04ab63f 100644
--- a/packer/scripts/kafka.sh
+++ b/packer/scripts/kafka.sh
@@ -8,6 +8,10 @@ kafka_version="2.0.0"
 
 ########################################
 
+echo "### Installing Kerberos Client..."
+
+sudo yum install -q -y krb5-workstation
+
 echo "### Downloading and installing Kafka version $kafka_version with Scala $scala_version..."
 
 kafka_name=kafka_${scala_version}-${kafka_version}
@@ -58,7 +62,7 @@ ExecStop=/opt/kafka/bin/kafka-server-stop.sh
 [Install]
 WantedBy=multi-user.target
 EOF
-sudo cp $systemd_tmp $systemd_kafka
+sudo mv $systemd_tmp $systemd_kafka
 sudo chmod 0644 $systemd_kafka
 
 systemd_connect=/etc/systemd/system/connect-distributed.service
@@ -83,7 +87,7 @@ ExecStop=/opt/kafka/bin/kafka-server-stop.sh
 [Install]
 WantedBy=multi-user.target
 EOF
-sudo cp $systemd_tmp $systemd_connect
+sudo mv $systemd_tmp $systemd_connect
 sudo chmod 0644 $systemd_connect
 
 systemd_zoo=/etc/systemd/system/zookeeper.service
diff --git a/packer/scripts/opennms.sh b/packer/scripts/opennms.sh
index dbcc109..2657ad9 100644
--- a/packer/scripts/opennms.sh
+++ b/packer/scripts/opennms.sh
@@ -3,9 +3,9 @@
 
 ######### CUSTOMIZED VARIABLES #########
 
-onms_repo="branches-features-sentinel"
+onms_repo="develop"
 onms_version="-latest-"
-grafana_version="5.2.2"
+grafana_version="5.2.4"
 
 ########################################
 
@@ -28,7 +28,6 @@ sudo sed -r -i '/name=Amazon Linux 2/a exclude=rrdtool-*' /etc/yum.repos.d/amzn2
 sudo yum install -y -q http://yum.opennms.org/repofiles/opennms-repo-stable-rhel7.noarch.rpm
 sudo rpm --import /etc/yum.repos.d/opennms-repo-stable-rhel7.gpg
 sudo yum install -y -q jicmp jicmp6 jrrd jrrd2 rrdtool 'perl(LWP)' 'perl(XML::Twig)'
-sudo yum install -y -q opennms-helm
 
 echo "### Installing OpenNMS..."
 
@@ -46,7 +45,9 @@ else
   echo "### Installing OpenNMS version $onms_version from $onms_repo Repository..."
   suffix="-$onms_version"
 fi
-sudo yum install -y -q opennms-core$suffix opennms-webapp-jetty$suffix opennms-webapp-hawtio$suffix
+sudo yum install -y -q opennms-core$suffix opennms-webapp-jetty$suffix
+sudo yum install -y -q opennms-helm$suffix
+sudo yum install -y -q opennms-webapp-hawtio$suffix
 
 echo "### Initializing GIT at $opennms_etc..."
 
diff --git a/packer/scripts/sentinel.sh b/packer/scripts/sentinel.sh
index 18f9da5..75c3c73 100644
--- a/packer/scripts/sentinel.sh
+++ b/packer/scripts/sentinel.sh
@@ -3,7 +3,7 @@
 
 ######### CUSTOMIZED VARIABLES #########
 
-sentinel_repo="branches-features-sentinel"
+sentinel_repo="develop"
 sentinel_version="-latest-"
 
 ########################################
@@ -17,9 +17,8 @@ echo "### Installing Common Packages..."
 sudo yum -y -q install haveged redis httpd
 sudo systemctl enable haveged
 
-echo "### Installing OpenNMS..."
+echo "### Installing OpenNMS Sentinel using $sentinel_repo repository..."
 
-echo "### Installing OpenNMSS $sentinel_repo Repository..."
 sudo yum install -y -q http://yum.opennms.org/repofiles/opennms-repo-$sentinel_repo-rhel7.noarch.rpm
 sudo rpm --import /etc/yum.repos.d/opennms-repo-$sentinel_repo-rhel7.gpg
 sudo yum install -y -q opennms-sentinel*
diff --git a/packer/sentinel.json b/packer/sentinel.json
index 79f5d06..c7f4d20 100644
--- a/packer/sentinel.json
+++ b/packer/sentinel.json
@@ -1,7 +1,7 @@
 {
   "variables": {
     "region": "us-east-2",
-    "source_ami": "ami-8c122be9",
+    "source_ami": "ami-0cf31d971a3ca20d6",
     "app_name": "OpenNMS Sentinel",
     "environment": "Drift"
   },
diff --git a/resources/minion/bootstrap-minion.sh b/resources/minion/bootstrap-minion.sh
index e6d0a72..c21e3bc 100644
--- a/resources/minion/bootstrap-minion.sh
+++ b/resources/minion/bootstrap-minion.sh
@@ -6,8 +6,13 @@ repo=${1-stable};
 version=${2--latest-};
 location=${3-Vagrant};
 opennms_url=${4-http://onmscore.aws.opennms.org:8980/opennms};
-kafka_svr=${5-kafka1.aws.opennms.org:9092,kafka2.aws.opennms.org:9092,kafka3.aws.opennms.org:9092};
-timezone=${6-America/New_York};
+kafka_servers=${5-kafka1.aws.opennms.org:9092,kafka2.aws.opennms.org:9092,kafka3.aws.opennms.org:9092};
+kafka_security_protocol=${6-SASL_PLAINTEXT};
+kafka_security_mechanism=${7-PLAIN};
+kafka_security_module=${8-org.apache.kafka.common.security.plain.PlainLoginModule};
+kafka_user_name=${9-opennms}
+kafka_user_password=${10-0p3nNMS};
+timezone=${11-America/New_York};
 
 # Internal Variables
 java_url="http://download.oracle.com/otn-pub/java/jdk/8u181-b13/96a7b8442fe848ef90c96a2fad6ed6d1/jdk-8u181-linux-x64.rpm"
@@ -146,13 +151,19 @@ http-url=$opennms_url
 EOF
 
   cat <<EOF > org.opennms.core.ipc.sink.kafka.cfg
-bootstrap.servers=$kafka_svr
+bootstrap.servers=$kafka_servers
 acks=1
+security.protocol=$kafka_security_protocol
+sasl.mechanism=$kafka_security_mechanism
+sasl.jaas.config=$kafka_security_module required username="$kafka_user_name" password="$kafka_user_password";
 EOF
 
   cat <<EOF > org.opennms.core.ipc.rpc.kafka.cfg
-bootstrap.servers=$kafka_svr
+bootstrap.servers=$kafka_servers
 acks=1
+security.protocol=$kafka_security_protocol
+sasl.mechanism=$kafka_security_mechanism
+sasl.jaas.config=$kafka_security_module required username="$kafka_user_name" password="$kafka_user_password";
 EOF
 
   cat <<EOF > org.opennms.netmgt.trapd.cfg
